bin.src/run_glance.py

```diff
@@ -10,7 +10,7 @@
 
 def setBatches(opsdb, colmap, args):
     bdict = {}
-    bdict.update(batches.glanceBatch(colmap, args.runName))
+    bdict.update(batches.glanceBatch(colmap, args.runName, sqlConstraint=args.sqlConstraint))
     return bdict
 
 
```

Comment:
At line 18 set subdir='glance' to match other run_batch scripts.

Reply:
This was already done? I'm a bit confused. However, good point. Done.